site/sfguides/src/root-cause-analysis-for-vehicle-product-quality-with-snowflake/root-cause-analysis-for-vehicle-product-quality-with-snowflake.md

```diff
@@ -0,0 +1,200 @@
+author: Kala Govindarajan
+id: root-cause-analysis-for-vehicle-product-quality-with-snowflake
+summary: 
+categories: Getting-Started,data-science-&-ml, solution-examples
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Cortex, Data Science & Ai
+
+# Root Cause Analysis for vehicle product quality with Snowflake
```

Comment:
@sfc-gh-kgovindarajan Change the title to "Root Cause Analysis for Vehicle Product Quality with Snowflake"

Reply:
changed the name